src/gui/tray/usermodel.h

```diff
@@ -200,8 +191,18 @@ class UserModel : public QAbstractListModel
     AccountAppList appList() const;
 
 signals:
-    Q_INVOKABLE void addAccount();
-    Q_INVOKABLE void newUserSelected();
+    void addAccount();
+    void currentUserChanged();
+
+public slots:
+    void fetchCurrentActivityModel();
+    void openCurrentAccountLocalFolder();
+    void openCurrentAccountTalk();
+    void openCurrentAccountServer();
+    void setCurrentUserId(const int &id);
+    void login(const int &id);
+    void logout(const int &id);
+    void removeAccount(const int &id);
```

Comment:
can you also remove all those useless `Q_INVOKABLE` macro from the `.cpp` file ?